forms.lua

```diff
@@ -1,99 +1,146 @@
+-- For backwards compability, check if 50 and 100 Minegeld notes exist
+local mg50_exists = minetest.registered_items["currency:minegeld_50"] ~= nil
+local mg100_exists = minetest.registered_items["currency:minegeld_100"] ~= nil
+
+-- Helper function to dynamically show or hide sections of a formspec
+-- if a currency item is missing
+local mg = function(count, str)
+	if count == 50 and mg50_exists then
+		return str
+	elseif count == 100 and mg100_exists then
+		return str
+	else
+		return ""
+	end
+end
+
```

Comment:
MG 100 was added in 2018, unnecessary IMO (https://github.com/mt-mods/currency/commit/03ad637fb10a139b97d90ac73690db81942a3919)